src/frontends/onnx/frontend/src/editor.hpp

```diff
@@ -29,11 +31,9 @@ class ONNX_IMPORTER_API ONNXModelEditor final {
     ///        is parsed and loaded into the m_model_proto member variable.
     ///
     /// \param model_path Path to the file containing the model.
-    ONNXModelEditor(const std::string& model_path,
-                    const std::shared_ptr<ov::frontend::TelemetryExtension>& telemetry = {});
+    ONNXModelEditor(const std::string& model_path, frontend::ExtensionHolder extensions = {});
 #if defined(OPENVINO_ENABLE_UNICODE_PATH_SUPPORT) && defined(_WIN32)
-    ONNXModelEditor(const std::wstring& model_path,
-                    const std::shared_ptr<ov::frontend::TelemetryExtension>& telemetry = {});
+    ONNXModelEditor(const std::wstring& model_path, ExtensionHolder extensions);
```

Comment:
without default value = {} ?

Reply:
It worked without a default value because there's no other use case currently. I've changed it though and added default initialization for the future.